digdag-plugin-utils/src/main/java/io/digdag/util/UserSecretTemplate.java

```diff
@@ -26,6 +32,13 @@ public boolean containsSecrets()
         return TEMPLATE_PATTERN.matcher(source).find();
     }
 
+    public void forEachKey(Consumer<String> action) {
```

Comment:
How about adding `List<String> getKeys()` instead of this? Because `Consumer` restricts use of checked exception. Users still can use `getKeys().stream().forEach(...)`.